test/main.js

```diff
@@ -260,6 +260,40 @@ describe('runSequence', function() {
 
 			called.should.eql(true);
 		})
+
+		it('should pass error if gulp execution halted in second execution', function(done) {
+			const stopTask = gulp.task('stopTask', function() {
```

Comment:
To ensure backwards compatibility, since this codebase is older, let's leave this as a `var`. (I struggle with it too, since I've been writing `const` and `let` for quite a while now.)